teachertool/src/types/errorCode.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorCode {
```

Comment:
Oh nice. We might want to strongly type logError to this. `logError(errorCode: ErrorCode,`